sources/packages/backend/libs/integrations/src/services/ssh/sftp-integration-base.ts

```diff
@@ -66,6 +66,112 @@ export abstract class SFTPIntegrationBase<DownloadType> {
     }
   }
 
+  /**
+   * Converts input into an ASCII 7 bit buffer.
+   * @param rawContent Raw content in string format.
+   * @returns a buffer of the input with extended ASCII accented characters converted to unaccented characters.
+   */
+  convertRawContent(rawContent: string) {
+    const content = Buffer.from(rawContent, FILE_DEFAULT_ENCODING);
```

Comment:
To attend the below AC the `FILE_DEFAULT_ENCODING` should be changed to `latin1`. The encoding change should not affect this method and will affect the [education-program-offering.institutions.controller.ts](https://github.com/bcgov/SIMS/blob/main/sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts#L516)
 [student-scholastic-standings.institutions.controller](https://github.com/bcgov/SIMS/blob/354171f89aeada90add4f58a4fae8f4c0d13e369/sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts#L220) allowing them to received extended ASCII ISO-8859-1.

Business AC:
_Ensure buffers converted as a result of a file uploaded are using latin1._

The files used for testing the above-mentioned API endpoint for offerings should also be saved in the correct encoding (ISO-8859-1). https://github.com/bcgov/SIMS/blob/46c9bfa3d7c399417bdb4f413251baa3e48d3a72/sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/bulk-insert

Reply:
updated